cmp/form/validation/constraints.js

```diff
@@ -23,8 +23,8 @@ import moment from 'moment';
 export const required = ({value, displayName}) => {
     if (
         isNil(value) ||
-        (isString(value) && value.trim().length == 0) ||
-        (isArray(value) && value.length == 0)
+        (isString(value) && value.trim().length === 0) ||
+        (isArray(value) && value.length === 0)
```

Comment:
Is the IDE inserting these `===` everywhere?  I don't really mind them, but curious as I've been seeing more and more of these diffs.

Reply:
no, but it is recommending them, and I am adding manually as appropriate.